electron_app/electron.js

```diff
@@ -109,34 +109,51 @@ Main._initializeApplication = function () {
     electron.ipcMain.on('app_ready', async(event) => {
       autoUpdater.autoDownload = false;
 
-      autoUpdater.checkForUpdates();
+      autoUpdater.checkForUpdates().then((result) => {
+        console.log(result.updateInfo.version);
 
-      const currentVersion = electron.app.getVersion();
-      const currentVersionIsPrerelease = prereleaseRegex.test(currentVersion);
-      autoUpdater.allowPrerelease = currentVersionIsPrerelease;
 
-      const downloadCancellationToken = new CancellationToken();
+        const currentVersion = electron.app.getVersion();
+        const currentVersionIsPrerelease = prereleaseRegex.test(currentVersion);
+        autoUpdater.allowPrerelease = currentVersionIsPrerelease;
 
-      console.log(`CurrentVersion: ${currentVersion}, CurrentVersionIsPrerelease: ${currentVersionIsPrerelease}`);
+        const downloadCancellationToken = new CancellationToken();
 
-      autoUpdater.addListener('error', (error) => {
-        event.sender.send('update_error');
-      });
+        console.log(`CurrentVersion: ${currentVersion}, CurrentVersionIsPrerelease: ${currentVersionIsPrerelease}`);
+
+        autoUpdater.addListener('error', (error) => {
+          event.sender.send('update_error');
```

Comment:
With consideration to the suggestion above:

```suggestion
          appReadyEvent.sender.send('update_error');
```